db/schema.rb

```diff
@@ -69,6 +69,7 @@
     t.index ["game_id"], name: "index_pieces_on_game_id"
   end
 
+
```

Comment:
Michelle, we should not edit the schema.rb file unless there is a migration to support the change. I know this is just a new line, but I don't want to risk messing with the database.